docs/ml_on_event_stop_detailed.md

```diff
@@ -0,0 +1,33 @@
+# on ML stop
+
+Start an [event handler](/reference/event-handler) (part of the program that will run when something happens). This handler works when when the ML model’s estimated action changes to the action you select.
+
+```sig
+ml.onStopDetailed(ml.event.Unknown, function (duration) {
+})
+```
+
+When an action changes, the stop event handler for the previous action will run, followed by the start event handler for the next action.
+
+For example, if your start event handler for an action starts music playing in the background, you could use a stop event handler to stop it.
+
+The event handler is passed a `duration` parameter. The duration is the number of milliseconds since this action became the estimated action. You can use the duration parameter in your code, for example displaying it or using a variable to keep a running total.
+
+## Parameters
+
+- **event**: one of the actions the machine learning model was trained on. The special value `unknown` represents the case where no action has a certainty above the recognition point.
+- **duration**: the [number](/types/number) of milliseconds the action was the estimated action.
```

Comment:
This is kinda weird as these are not two parameters to the same function. Event is a parameter in the usual way. Duration is a parameter passed to the user callback.

Reply:
Maybe split into "Parameters" and "Event handler parameters" if we can't find a MakeCode convention for this?

Reply:
Or we drop `duration` because the paragraph above the "Parameters" section is really clear and covers the same ground. Tempted by this.

Reply:
For radio top-level blocks, MakeCode include the callback values as parameters, but, they don't have the two types of parameters that we do.

![image](https://github.com/user-attachments/assets/0622cc31-a92e-438a-8310-d159c054feaa)

"Event handler parameters" or similar do not appear to be used elsewhere. 

I think I'm also in favour of dropping duration from the parameters section and rely on the previous paragraph.

Reply:
[c54d903](https://github.com/microbit-foundation/pxt-microbit-ml/pull/29/commits/c54d903ad6b35d78bb5430f0adfdd067775ae3f3)